src/translations/en.ts

```diff
@@ -323,6 +323,7 @@ const en = {
     },
 
     card: {
+      card_id_title: 'Card ID',
```

Comment:
nit: but why not just `card_id` ?

Reply:
Its companion is `card_title`. Just following that pattern